src/test/java/com/plaid/client/integration/ItemAddTokenCreateTest.java

```diff
@@ -37,4 +37,25 @@ public void testSuccess() throws Exception {
     assertNotNull(response.body().getAddToken());
     assertTrue(response.body().getExpiration().after(new Date()));
   }
+
+  @Test
+  public void testSuccessWithUserOptions() throws Exception {
+    String clientUserId = Long.toString((new Date()).getTime());
+    String legalName = "John Doe";
+    String phoneNumber = "+1 415 555 0123";
+    String emailAddress = "example@plaid.com";
+    Date verifiedTime = new Date();
+    ItemAddTokenCreateRequest.User user =  new ItemAddTokenCreateRequest
+      .User(clientUserId)
+      .withLegalName(legalName)
+      .withVerifiedPhoneNumber(phoneNumber, verifiedTime)
+      .withEmailAddress(emailAddress);
```

Comment:
Wondering if we should split this into 2 tests - one for withEmailAddress/withPhoneNumber and another for the verified variants.

Reply:
Good call -- will do!

Reply:
⚡